Doc/library/xml.rst

```diff
@@ -63,19 +63,21 @@ the various modules are vulnerable to them.
 =========================  ==============   ===============   ==============   ==============   ==============
 kind                       sax              etree             minidom          pulldom          xmlrpc
 =========================  ==============   ===============   ==============   ==============   ==============
-billion laughs             **Vulnerable**   **Vulnerable**    **Vulnerable**   **Vulnerable**   **Vulnerable**
+billion laughs             Safe (1)         Safe (1)          Safe (1)         Safe (1)         Safe (1)
```

Comment:
I'm against marking it safe until Python no longer supports libexpat <= 2.4.0.

Reply:
That's a fair point. Any ideas how to best communicate it in this table?

Reply:
Vulnerable or Safe depends on the libexpat version, that's what I wrote in the footnote (1). I explain how how to check manually if your Python is vulnerable or not.

@tiran How do you want to explain that it depends on the libexpat version in this table, if you are unhappy with "Safe (1)"?

Reply:
Do ``**Vulnerable** (1)`` until all relevant Linux distros have fixed libexpat: all supported CentOS streams, Debian stables, RHELs, Ubuntu LTS, etc.